packages/hooks/src/useHasFiatCurrency.ts

```diff
@@ -0,0 +1,12 @@
+import { useStore } from '@deriv/stores';
+
+const useHasFiatCurrency = () => {
+    const { client } = useStore();
+    const { account_list, is_crypto } = client;
+
+    const has_fiat_currency = account_list.some(account => account.title !== 'Real' && !is_crypto(account.title));
```

Comment:
I'm a bit confused by `account.title !== 'Real'` 🤔 Isn't `title` always a currency like `BTC` or `USD`?

Reply:
It will return both USD, BTC and Real but for VRTC login id
<img width="215" alt="Screenshot 2023-04-12 at 4 54 37 PM" src="https://user-images.githubusercontent.com/103412909/231406131-b553c0a5-67e4-44bc-b029-33110a394f41.png">

Reply:
@ameerul-deriv Oh boy :)))) Real is Virtual? :D

![blow-mind-mind-blown](https://user-images.githubusercontent.com/72082844/231418961-b04f710c-768c-473f-a44c-4063bc965813.gif)

Now the existing code makes sense, Thank you for lighting me up man 😅